AsyncDisplayKit/Details/UIView+ASConvenience.h

```diff
@@ -42,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)setNeedsDisplay;
 - (void)setNeedsLayout;
```

Comment:
Where should they be or do you think we should add specific documentation about them?